UsageControlFramework/src/test/java/it/cnr/iit/usagecontrolframework/rest/UsageControlFrameworkScenarioIntegrationTest.java

```diff
@@ -83,7 +86,7 @@ public void an_endAccess_request_preceeding_startAccess_sends_PEP_endAccessRespo
     }
 
     @Test
-    public void a_tryAccess_request_sends_PEP_tryAccessResponse_containg_Deny_decision() {
+    public void a_tryAccess_request_sends_PEP_tryAccessResponse_containg_Deny_decision() throws URISyntaxException, IOException {
```

Comment:
No, Tests should never throw exception.
They should catch them in the stages and gracefully end with fail()
Or if a test is expecting an exception then it should be asserted at the ThenSatge 
(example PEPRest_service_$_execution_fails)

Reply:
ok! I'll just catch the exception inside if it fails everything will fail anyway.